src/pages/ValidateLoginPage.js

```diff
@@ -0,0 +1,33 @@
+import {Component} from 'react';
+import PropTypes from 'prop-types';
+import lodashGet from 'lodash/get';
+import routePropTypes from './routePropTypes';
+import {validateLogin} from '../libs/actions/User';
+
+const propTypes = {
+    /* Onyx Props */
+
+    // The accountID and validateCode are passed via the URL
+    route: PropTypes.objectOf(routePropTypes),
+};
+
+const defaultProps = {
+    route: {
+        params: {},
+    },
+};
+
+export default class ValidateLoginPage extends Component {
+    componentDidMount() {
+        const accountID = lodashGet(this.props.route.params, 'accountID', '');
+        const validateCode = lodashGet(this.props.route.params, 'validateCode', '');
+        validateLogin(accountID, validateCode);
+    }
+
+    render() {
+        return null;
```

Comment:
Can you please add a comment here to explain why nothing is being rendered (just explain that this page is handling the validate route and will redirect to the proper page after it's done)?